client/v3/balancer/balancer.go

```diff
@@ -138,6 +138,18 @@ type baseBalancer struct {
 	picker picker.Picker
 }
 
+// UpdateClientConnState implements "grpc/balancer.Balancer" interface.
+func (bb *baseBalancer) UpdateClientConnState(ccs balancer.ClientConnState) error {
+	bb.HandleResolvedAddrs(ccs.ResolverState.Addresses, nil)
+	// TODO: This **should** incorporate the lines that warn in `HandleResolvedAddrs`.
```

Comment:
Note, this `TODO` should almost certainly be resolved before merging.

Reply:
E.g. 

- https://github.com/etcd-io/etcd/blob/5dcd459ae9c7948f5620002f5b0bb9cf0b8f1502/client/v3/balancer/balancer.go#L145 should be returned here as an `error`

I **can** just rename / repurpose `HandleResolvedAddrs()` here since it is no longer in the `Balancer` interface:

- https://pkg.go.dev/google.golang.org/grpc@v1.29.1/balancer#Balancer
- https://pkg.go.dev/google.golang.org/grpc@v1.35.0/balancer#Balancer